style.css

```diff
@@ -6,14 +6,14 @@
   body {
     font-family: 'Poppins', serif;
     box-sizing: border-box;
-    height: 100vh;
+    height: 718px;
     color: #172b4d;
     margin: 0;
     padding: 0;
-    width: 100vw;
+    width: 375px;
```

Comment:
- [ ] [Optional] To have a proper responsive design please avoid using fixed width and height values for your project. This will complicate things a lot in the future for you